flink-ml-lib/src/test/java/org/apache/flink/ml/evaluation/BinaryClassificationEvaluatorTest.java

```diff
@@ -297,14 +298,20 @@ public void testEvaluateWithMultiScore() {
     public void testEvaluateWithWeight() {
         BinaryClassificationEvaluator eval =
                 new BinaryClassificationEvaluator()
-                        .setMetricsNames(BinaryClassificationEvaluatorParams.AREA_UNDER_ROC)
+                        .setMetricsNames(
+                                BinaryClassificationEvaluatorParams.AREA_UNDER_ROC,
+                                BinaryClassificationEvaluatorParams.AREA_UNDER_PR)
                         .setWeightCol("weight");
         Table evalResult = eval.transform(inputDataTableWithWeight)[0];
-        List<Row> results = IteratorUtils.toList(evalResult.execute().collect());
+        Row result = (Row) IteratorUtils.toList(evalResult.execute().collect()).get(0);
         assertArrayEquals(
-                new String[] {BinaryClassificationEvaluatorParams.AREA_UNDER_ROC},
+                new String[] {
+                    BinaryClassificationEvaluatorParams.AREA_UNDER_ROC,
+                    BinaryClassificationEvaluatorParams.AREA_UNDER_PR
+                },
                 evalResult.getResolvedSchema().getColumnNames().toArray());
-        assertEquals(EXPECTED_DATA_W, results.get(0).getFieldAs(0), EPS);
+        assertArrayEquals(
+                EXPECTED_DATA_W, new double[] {result.getFieldAs(0), result.getFieldAs(1)}, 1e-9);
```

Comment:
nit: Let's use `EPS` instead of `1e-9` for consistency with existing tests.

Reply:
Thanks. I've update this line.